### PR TITLE
[14.0] [FIX] sale_coupon_delivery_auto_refresh: duplicate sale order

### PR DESCRIPTION
**Pull request by renda-dev**
Remove coupon and delivery lines from sale order before duplicating, otherwise it would raise an error.

I'm also proposing me and my team as maintainers of this module.

### CODE REVIEW

**Review by HekkiMelody — APPROVED** [review]
Code review, LGTM

**Review by francesco-ooops — APPROVED** [review]
Functional ok!

@pedrobaeza could you please merge? (it includes maintainership)

File: sale_coupon_delivery_auto_refresh/__manifest__.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
     "version": "14.0.1.0.0",
     "summary": "Allows to auto-apply the coupons after adding delivery line",
     "author": "Ooops404, Cetmix, , Odoo Community Association (OCA)",
```
Comment (pedrobaeza) [inline sale_coupon_delivery_auto_refresh/__manifest__.py]:
```suggestion
    "author": "Ooops404, Cetmix, Odoo Community Association (OCA)",
```

Comment (renda-dev) [reply sale_coupon_delivery_auto_refresh/__manifest__.py]: Fixed, thank you!

**Pull request merged**